playbooks/roles/uptime_report/defaults/main.yml

```diff
@@ -0,0 +1,24 @@
+#
+# Open edX Retirement Pipeline Configuration
+#
+# github:     https://github.com/edx/configuration
+# wiki:       https://openedx.atlassian.net/wiki/display/OpenOPS
+# code style: https://openedx.atlassian.net/wiki/display/OpenOPS/Ansible+Code+Conventions
+# license:    https://github.com/edx/configuration/blob/master/LICENSE.TXT
+#
+#
+# Deploy uptime_report
+#
+# See documentation in README.rst
+
+#
+# This file contains the variables you'll need to pass to the role, and some
+# example values.
+
+uptime_report_script: "/edx/var/upload_uptime_report.py"
+uptime_report_cron_job_days: 6
+uptime_report_cron_job_minutes: 0
+newrelic_monitor_id: ""
```

Comment:
@mavidser Would this work for new instances or only redeployments? Monitor id gets generated as part of instance deployment right? Is there a way to get this automatically?

Reply:
Good catch - monitor id changes whenever the appserver URL changes, which does happen. Using monitorName now.